src/nhp/docker/__main__.py

```diff
@@ -15,9 +15,11 @@
 from azure.storage.blob import BlobServiceClient
 from azure.storage.filedatalake import DataLakeServiceClient
 
-import config
-from model.helpers import load_params
-from run_model import run_all
+from nhp.docker import config
+from nhp.model.__main__ import (
+    run_all,  # FIXME: Not a good idea to import from __main__.py
```

Comment:
good point - should refactor the `nhp/model/__main__.py` to extract out all of the useful functions. it was kept separate previously because there are a couple of functions which are really only useful for debugging/testing.

Perhaps create `nhp/model/run.py`, move all the functions out of main except for the `main()`, `_run_model_argparser()` [that should be renamed I guess!]